src/web/make-report.rkt

```diff
@@ -67,12 +67,9 @@
   (cond
    [(not dir) (void)]
    [(> (length pareto-points) 1) ; generate the scatterplot if necessary
-    (call-with-output-file (build-path dir "cost-accuracy.png")
+    (call-with-output-file (build-path dir "cost-accuracy.json")
       #:exists 'replace
-      (λ (out) (make-full-cost-accuracy-plot pareto-max pareto-start pareto-points out)))]
-   [else
-    (when (file-exists? (build-path dir "cost-accuracy.png"))
-      (delete-file (build-path dir "cost-accuracy.png")))])
+      (λ (out) (make-full-cost-accuracy-json pareto-max pareto-start pareto-points out)))])
```

Comment:
Is this json being removed in the next cleanup PR?

Reply:
Yes, I'm working on it right now